tailwind.config.ts

```diff
@@ -37,6 +37,12 @@ export default {
       semibold: "600",
     },
     fontSize: {
+      xs: [
```

Comment:
[This one](https://www.figma.com/design/b0A4ttd9C576bp2roJh0hX/Foundations?node-id=1151-288&t=aP6VTqePW8zZAVrS-4) was missing or added before, so adding it here as we need it for this component.